slovenian/začetek.md

```diff
@@ -0,0 +1,9 @@
+Včeraj si zaspal v svoji sobi, zdaj pa si se zbudil v beli sobi z dvemi vrati. Ena na levi, druga na desni.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.